src/app/search/search-filters/known-values.tsx

```diff
@@ -181,7 +180,15 @@ const knownValuesFilters: FilterDefinition[] = [
     keywords: ['craftable'],
     description: tl('Filter.Craftable'),
     destinyVersion: 2,
-    filter: () => (item) => craftableHashes.includes(item.hash),
+    filter: () => (item) => item.crafted,
```

Comment:
This change seems wrong.

Reply:
Good catch, it's part of a revert that I forgot to finish. We don't need craftableHashes anymore since we should be able to use the presence of the `patternUnlockRecord`.